modules/op-release-notes-1-8.adoc

```diff
@@ -31,10 +31,14 @@ This change affects the pipeline component, which includes the `TaskRun`, `Pipel
 Bug fixes and support for the `tekton.dev/v1alpha1` API version are provided only through the end of the current GA 1.8 lifecycle. Use the `tekton.dev/v1beta1` API version instead.
 //  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
 
+* In {pipelines-title} 1.8, support for the `PipelineResource` custom resource (CR) has been removed. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
```

Comment:
Support is being removed or the CRD is being removed altogether?

Reply:
> Support is being removed or the CRD is being removed altogether?

@vdemeester ?

Reply:
The CRD is still there, but it is officially mark as deprecated upstream, hence "support" being removed.

Reply:
We had already marked PipelineResource and Condition CRDs as deprecated in 1.3 and 1.2 respectively which is basically the same as what these bullet points is conveying. If we know in which release they will be removed altogether, then we should add it to the release notes and give a heads-up to the customers. Otherwise I'm not sure we need to reiterate the upstream deprecation since downstream they were already deprecated.